user/harec/patches/link-cflags.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/Makefile b/Makefile
+index a7b4ed3..fe27227 100644
+--- a/Makefile
++++ b/Makefile
+@@ -36,7 +36,7 @@ harec_objects = \
+ $(BINOUT)/harec: $(harec_objects)
+ 	@mkdir -p -- $(BINOUT)
+ 	@printf 'CCLD\t%s\n' '$@'
+-	@$(CC) $(LDFLAGS) -o $@ $(harec_objects) $(LIBS)
++	@$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $(harec_objects) $(LIBS)
```

Comment:
we have a qbe patch doing the same thing so i did it here as well, but not sure if it has any effect on the build

Reply:
i think this was actually a bug workaround back in like llvm 14-16 and is not actually required anymore and yields an identical (i.e. bit for bit) result as the individual .o's have their flags stored to use for lto and whatever